core/lib/page-functions.js

```diff
@@ -265,7 +265,7 @@ function getNodePath(node) {
     }
     let index = 0;
     let prevNode;
-    while (prevNode = node.previousSibling) {
+    while (prevNode = node.previousSibling) { // eslint-disable-line no-cond-assign
```

Comment:
If we're going to enable this rule, we should try and stick to it.

There are other ways of organizing this code.